pkg/gds/admin.go

```diff
@@ -785,7 +780,10 @@ func (s *Admin) ListVASPs(c *gin.Context) {
 			snippet.Name, _ = vasp.Name()
 
 			// Add verified contacts to snippet
-			snippet.VerifiedContacts = models.ContactVerifications(vasp)
+			if snippet.VerifiedContacts, err = models.ContactVerifications(vasp); err != nil {
+				log.Warn().Err(err).Msg("could not get contact verifications")
+				c.JSON(http.StatusInternalServerError, admin.ErrorResponse(err))
+			}
```

Comment:
Instead of returning a 500 here, let's optimistically try to return the verified contacts we do have, and log the error:

```suggestion
			if snippet.VerifiedContacts, err = models.ContactVerifications(vasp); err != nil {
				log.Error().Err(err).Msg("could not get all verified contacts")
			}
```